config-linux.md

```diff
@@ -549,6 +549,7 @@ The following parameters can be specified for the container:
 
     * If `closID` is set, and neither of `l3CacheSchema` and `memBwSchema` are set, runtime MUST check if corresponding pre-configured directory `closID` is present in mounted `resctrl`. If such pre-configured directory `closID` exists, runtime MUST assign container to this `closID` and [generate an error](runtime.md#errors) if directory does not exist.
 
+* **`monitoring`** *(boolean OPTIONAL)* - specifies if monitoring features (eg. MBM, CMT) should be enabled
```

Comment:
How about:
"specifies if Intel RDT monitoring features (CMT and MBM) should be enabled."?